Portainer add update platform (#43706)
home-assistant/home-assistant.github.io: source/_integrations/portainer.markdown

- **Data attribute**: `dangling`
    - **Description**: If true, only prune dangling images.
    - **Optional**: Yes
 
## Update

> The integration utilizes a watcher, that runs every three hours in the backgrouond. This will check if the current installed images against the latest tag in the repositories.

Reviewer: ```suggestion
The integration checks for updates every 3 hours.
```
Would this be sufficient? Or is it important to know that it is a background job that checks the tags?

Author: I suppose so, but that's more from a developers perspective. The user won't see anything of it on the frontend, expect when there's an update. Otherwise it's just a background task crunching its tasks.

Reviewer: Ah ok. Since this is the user documentation, I would remove the detailed description and just mention how often it checks for updates.

> If an update is available, this will be shown in the Config Updates. When updating, the integration will initiate a recreate of the container and install the latest tag.

Reviewer: Config Updates, is this here, in Home Assistant? {% my updates title="**System** > **Updates**" %}?

Author: That'd be correct.

> If an update is available, this will be shown in the Config Updates. When updating, the integration will initiate a recreate of the container and install the latest tag.

Reviewer: ```suggestion
If an update is available, this will be shown under {% my updates title="System > Updates" %}. When updating, the integration will recreate the container and install the latest tag.
```
